doc/ChangeLog

```diff
@@ -1,4 +1,266 @@
+One-line Summary: Bring in several fixes for testing in the previous cesm3_0_beta03/04 tags
+
+Purpose and description of changes
+----------------------------------
+
+Fix a list of issues mostly for testing that came in with the cesm3_0_beta03 and cesm3_0_beta04 tags
+for the science "chill" deadline bringing in the baseline science capabilities needed for the cesm3_0 release.
+
+List of things:
+
+- New polarcap grid
+- Fix use_init_interp for several resolutions that had
+- Remove clm5_1 physics option
+- Newly needed surface datasets added to auto build in Makefile for mksurfdata_esmf
+- Fix several individual tests that were failing
+- Update to submodules to ones roughly based on cesm3_0_beta04
+- Add graceful error checking to the FATES parameter file tests that will get it working in some cases
+
+Contributors
+------------
+    @slevis-lmwg @ekluzek @adamrher
+
+Significant changes to scientifically-supported configurations
+--------------------------------------------------------------
+
+Does this tag change answers significantly for any of the following physics configurations?
+(Details of any changes will be given in the "Answer changes" section below.)
+
+    [Put an [X] in the box for any configuration with significant answer changes.]
+
+[ ] clm6_0
+
+[ ] clm5_0
+
+[ ] ctsm5_0-nwp
+
+[ ] clm4_5
+
+
+Bugs fixed
+----------
+List of CTSM issues fixed (include CTSM Issue # and description) [one per line]:
+
+  #2861 -- Nag compiler mpi-serial tests are failing on Izumi after the OS upgrade
+  #2801 -- fsurdat file needed for NEON MOAB site bfb done testing
+  #2791 -- Add f19 16pft 1850 and Hist to list of surface dataset resolutions (for the PPE work)
+  #2780 -- New CN matrix fails with single point sites with the new ctsm5.3 datasets.
+  #2654 -- Failing tests for izumi_nag with mpi-serial in cesm3_0_alpha02a
+  #2640 -- Update submodules to cesm3_0_beta04 versions
+  #2548 -- ne0ARCTICne30x4 grid transient failure
+  #2544 -- Failing ne0CONUSne30x8_ne0CONUSne30x8_mt12 in CESM testing
+  #2486 -- Temporarily add back a T42 dataset for CAM
+  #2379 -- Deprecate clm5_1 physics with ctsm5.2.0
+  #2294 -- use of mct_mod is deprecated
+
+Notes of particular relevance for users
+---------------------------------------
+Changes to CTSM's user interface (e.g., new/renamed XML or namelist variables):
+  Removal of the clm5_1 physics option to CLM_PHYSICS_VERSION XML variable
+
+Changes made to namelist defaults (e.g., changed parameter values):
+  Add support for new grid: ne0np4.POLARCAP.ne30x4
+  Add support for f19 and f45 16-pft surface datasets
+  Fix some finidat issues for: f19, VR grids
+  Remove more mention of vichydro in namelist defaults
+
+Notes of particular relevance for developers:
+---------------------------------------------
+
+Caveats for developers (e.g., code that is duplicated that requires double maintenance):
+   I think we should expand the shell_commands unit testing I added in FatesSetupParam.
```

Comment:
@glemieux I'd like to get your thoughts on this section I have here. I think we should meet to discuss it a bit, and talk about it in an upcoming CTSM SE meeting.

FYI: I'll probably resolve this conversation before you get a chance to look at it, just to finish the PR.